.lintstagedrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
It may not be needed but I like being explicit when dealing with commonjs vs esmodules since it can be a bit of a clustergoof. Would you prefer the file be named `.lintstagedrc.js` if it doesn't seem to break anything?

Reply:
I was commenting about adding a lint filter for `cjs` and not the filename.

Reply:
Ah, my mistake. As we discussed on Teams, we would need to add a tsconfig and eslintrc to the root  in order to lint `.lintstagedrc.cjs` and I'm not sure if that would conflict with the package-specific linting setups. 

The next step in the monorepo setup is splitting out the shared eslint configuration and extending it in the individual packages. Once that happens we use the shared eslintrc in the root directory, and include the cjs file. 

Here is one of turbo's examples that shows this in practice:
https://github.com/vercel/turbo/tree/main/examples/design-system

We held off on splitting code out of CC for now becaues it makes more sense to make the monorepo transition incrementally. I'll create a follow up issue for splitting out the eslint/tsconfig/storybook/demos.

Reply:
#7122